boards/arm/nrf52820dongle_nrf52820/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+zephyr_library()
+zephyr_library_sources(board.c)
+zephyr_library_include_directories(${ZEPHYR_BASE}/drivers)
```

Comment:
What do you actually need this line for?

Reply:
another copy paste from the 840 dongle - will remove it unless something complains

Reply:
done